recipes/qsum/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "qsum" %}
+  {% set version = "0.2.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/QCoding/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 'e5c01285550d9df96568b12e711d52074263d42f0ba4078dd3fd4aff69c201f6'
+
+build:
+  noarch: python
+  number: 0
+
```

Comment:
For `noarch` packages, you can remove the `bld.bat` and `build.sh` files, and add `script: "{{ PYTHON }} -m pip install . -vv"` here in the `script` section, so please do that.